rule/early_return.go

```diff
@@ -1,3 +1,4 @@
+// Package rule implements revive's linting rules.
```

Comment:
yes because we have 83 files in rule directory, and I thought that is a good issue to update that :)

Reply:
This line is not needed to be duplicated. The only place where it should be is https://github.com/mgechev/revive/blob/655e6060b2ef3915215e0d795c79131f1a48d7f1/rule/doc.go#L1 See https://go.dev/doc/comment for details.

Because we will have the following in the [doc](https://pkg.go.dev/github.com/mgechev/revive/rule):
<img width="1123" alt="image" src="https://github.com/user-attachments/assets/6ba7dbf7-cda5-47b1-9a6a-d18c5f05c4e4">

Reply:
ok removed all package comments except doc.go :)